authentication/authentication-common/src/test/java/io/mosip/authentication/common/service/transaction/manager/IdAuthSecurityManagerTest.java

```diff
@@ -300,8 +309,97 @@ public void hashTest() throws IdAuthenticationBusinessException {
 		String id = "12";
 		Mockito.when(uinHashSaltRepo.retrieveSaltById(Mockito.any())).thenReturn(id);
 		String actualResponse = "CBFAD02F9ED2A8D1E08D8F74F5303E9EB93637D47F82AB6F1C15871CF8DD0481";
+		Mockito.when(identityRepo.existsById(Mockito.anyString())).thenReturn(true);
+		String response = authSecurityManager.hash(id);
+		assertEquals(response, actualResponse);
+	}
+	
+	@Test(expected = IdAuthenticationBusinessException.class)
+	public void hashTest_salt_keu_not_exists() throws IdAuthenticationBusinessException {
+		String id = "12";
+		Mockito.when(uinHashSaltRepo.retrieveSaltById(Mockito.any())).thenReturn(null);
+		String actualResponse = "CBFAD02F9ED2A8D1E08D8F74F5303E9EB93637D47F82AB6F1C15871CF8DD0481";
+		Mockito.when(identityRepo.existsById(Mockito.anyString())).thenReturn(true);
+		String response = authSecurityManager.hash(id);
+		assertEquals(response, actualResponse);

```

Comment:
if the expectation is exception, why do we need assert equals here ?

Reply:
Lets check this for other test cases with exception expectation also

Reply:
Done